### ISSUE TITLE
Remove unused constant and console.log

### ISSUE THREAD

**Issue by ElmoArmy** [yam-finance/synths-sdk#103]
Eslint shows an issue for the following line. @byterose As the author of this function, you see any reason why we shouldn't remove this?

@typescript-eslint/no-unused-vars
https://github.com/yam-finance/synths-sdk/blob/0bf7dbf3b51334a718b58c71e44e80c82e330c20/src/lib/Synths.ts#L72

Suggestion would be to replace that line with ","  effectively ignoring that parameter during deconstruction. 

I also propose removing the console statement:
https://github.com/yam-finance/synths-sdk/blob/0bf7dbf3b51334a718b58c71e44e80c82e330c20/src/lib/Synths.ts#L79

let me know what you think

**Comment by byterose** [comment]
@ElmoArmy Sorry, still working on that function ^^

**Comment by ElmoArmy** [comment]
> @ElmoArmy Sorry, still working on that function ^^

Ok cool, well I'll let you close this issue when it is ready.

**Comment by byterose** [comment]
I removed the console.log but can't remove the unused constant so i ignored it for now. Any ideas how to solve this without changing the underlying helper function?

**Comment by ElmoArmy** [comment]
> I removed the console.log but can't remove the unused constant so i ignored it for now. Any ideas how to solve this without changing the underlying helper function? @ElmoArmy

I think I could have been more clear. Let me try again. To ignore a variable when deconstructing an array, do the following in your code:
![image](https://user-images.githubusercontent.com/1284031/144128668-5d0d9749-74dc-4d68-9429-f499ae931159.png)

https://www.samanthaming.com/tidbits/13-skip-values-in-destructuring/
